ast.go

```diff
@@ -145,7 +145,12 @@ func (fa *ASTFunc) generateParameters() []ast.Expr {
 		if p.Type.IsSlice && !p.Type.IsReturnArgument {
 			hasDeclaration = true
 
-			fa.addGoToCSliceConversion(p.Name, p.Type)
+			
+			if _, ok := p.Heuristics["pass_c"]; !ok {
```

Comment:
I seriously have no clue whatsoever anymore what my intention with the Heuristics map was